src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -450,6 +453,45 @@ void assertLogMatches(WorkflowRun build, String regexp) throws IOException { //
         });
     }
 
+    @Test public void detailsExported() throws Exception {
+        story.addStep(new Statement() {
+            @Override
+            public void evaluate() throws Throwable {
+                DumbSlave s = story.j.createOnlineSlave();
+
+                JSONParser parser = new JSONParser();
+                JSONObject propertiesJSON = null;
```

Comment:
nit: No need to declare and initialize the variable to null up here, you can just declare and assign it directly below where it is used.